plutus-pab/demo/pab-nami/README.md

```diff
@@ -0,0 +1,66 @@
+3. The frontend application calls the `PayToWallet` contract PAB endpoint with the recipient's payment key hash and stake key hash, and the amount to send in Lovelace.
+4. The PAB constructs a partial/unbalanced transaction and makes it available through it's contract instance's status endpoint.
+5. The frontend application fetches this partial/unbalanced transaction. It then balances it and signs it using the Nami wallet api.
+6. Finally, the frontend application submits the final transaction using the Nami wallet api (which in turn, uses the blockfrost API).
+
+## Run the demo
+
+The instructions were tested inside the `plutus-apps`'s `nix-shell`.
+
+The demo contains two parts: the PAB application in `plutus-pab/demo/pab-nami/pab` and the frontend application in `plutus-pab/demo/pab-nami/client` which interacts with the PAB and the Nami wallet.
+
+### Setup Nami wallet
+
+1. Install the Nami wallet browser extension (currently, the Nami wallet is only available in Chrome-based browsers)
+2. Setup your wallet password and seed phrase as required.
+3. Create a second account in your wallet in order to test the demo.
+4. In Nami wallet's `Settings > Network`, select the Cardano testnet.
+5. Add some funds to your main account using the [Cardano faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/) (when you have finished using your test tokens, please return them to the faucet following the instructions in their website).
+6.
+
+### Run the PAB
+
+From a command line interface, make sure you're in the root folder of the `plutus-apps` repository, and run the following commands:
+
+```
+# 'Migrate' command to initialise the PAB database.
+$ cabal run plutus-pab:exe:plutus-pab-nami-demo -- migrate --config plutus-pab/demo/pab-nami/pab/config.yaml
+
+# Run the PAB webserver
+$ cabal run plutus-pab:exe:plutus-pab-nami-demo -- webserver --config plutus-pab/demo/pab-nami/pab/config.yaml
+```
+
+### Run the demo frontend
+
+From another command line interface, run the following command to launch the frontend application:
+
+```
+$ cd plutus-pab/demo/pab-nami/client
+
+# To install NPM dependencies
+$ npm install
+
+# To install Spago dependencies
+$ spago install
+
+# Run the dev server
+$ npm run build:webpack:dev
+```
+
+Open the browser with the frontend application's URL (`http://localhost:8009`). You will see two fields: the recipient's Bech32 Cardano address and an amount to send to it in Lovelace. From the Nami wallet interface, change to your *second* account, click on the `Receive` tab, copy the shown Cardano address and paste it in the application's form. Then, go back to your *first* account which contains your funds. Choose a lovelace amount (minimum of 2_000_000 Lovelace) and click on `Make payment`. The application should show the transaction id that was submitted to the Cardano testnet. After waiting for a bit, you should see the funds change in Nami wallet's interface.
```

Comment:
On navigating to http://localhost:8009 for the first time you will see Nami popup requesting access. Once "Access" is clicked then the header and fields are populated.

Ideally, before connecting it'd say "Connect to Nami" instead of blank white page.

Reply:
Done!